docs/basic-features/data-fetching/incremental-static-regeneration.md

```diff
@@ -27,6 +27,8 @@ description: 'Learn how to create or update static pages at runtime with Increme
 
 Next.js allows you to create or update static pages _after_ you’ve built your site. Incremental Static Regeneration (ISR) enables you to use static-generation on a per-page basis, **without needing to rebuild the entire site**. With ISR, you can retain the benefits of static while scaling to millions of pages.
 
+> Note: the `experimental-edge` runtime (https://nextjs.org/docs/api-reference/edge-runtime) is currently not compatible with ISR although can leverage `stale-while-revalidate` by setting the `cache-control` header manually.
```

Comment:
I think this could be a bit confusing: [ISR isn't just caching](https://vercel.com/docs/concepts/next.js/overview#incremental-static-regeneration)!

> **Note:** The [Edge Runtime](https://nextjs.org/docs/api-reference/edge-runtime) is not currently compatible with ISR.

This note is also pretty high up on the page for something I don't think most people need to know.

Reply:
Hmm yeah it's not just caching although the cache-control header provides an alternative in the mean-time while it's implemented and I don't think we want to burry this note for users who are confused by this. 

We potentially could make this a build time warning instead.

Reply:
Build time warning makes sense to me 👍

Reply:
What does "not currently compatible" look like? Will it just never update? Will it error? Will it do nothing?